<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPumpTankData</class>
 <widget class="QWidget" name="FormPumpTankData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="buttonWebsitePump">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>530</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>网址收集</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFilePump">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>580</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>文件打包收集</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditWebsite">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTankNameFile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>输出文件：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>收集进程：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>坦克名称文件：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutFile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditProcess">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>631</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>坦克网址：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
